README.md

```diff
@@ -19,7 +19,7 @@ Some more details are covered in this YouTube video: [Operating FoundationDB on
 ## Running the Operator
 
 To run the operator in your environment, you need to install the operator and the CRDs:
-*Note* this will install the latest version from main.
+_Note_ this will install the latest version from main.
```

Comment:
Prettier made some formatting changes. Let me know if you want me to remove them from this PR.